setup/centos5/ansible-playbook.yaml

```diff
@@ -1,35 +1,27 @@
 ---
 
 - hosts: iojs-build-centos5
-  # This first set of tasks is to simply get python-simplejson onto
-  # the EL5 machine because it's needed by Ansible when using
-  # older versions of Python on the managed node
-
   gather_facts: False
 
-  remote_user: root
-
   tasks:
+    - name: Bootstrap | Check python version
+      command: python -V
+      register: python_version
 
-    - include_vars: ansible-vars.yaml
-      tags: vars
-
-    - name: General | Bootstrap Python for Ansible
-      raw: yum -y install python-simplejson
-      tags: general
-
+    - name: Bootstrap | Install ansible requirements
+      yum: name=python-simplejson state=latest
+      when: python_version.stderr|search("2.4")
```

Comment:
Just fixed the yum invoke; sorry about that. Probably fiddled around after python was installed. Strange that ansible puts it in stderr when used while raw uses stdout. Oh well.